--- origsrc/cracklib-2.8.15/lib/Makefile.am	2009-11-18 17:58:20.000000000 -0600
+++ src/cracklib-2.8.15/lib/Makefile.am	2010-02-16 14:25:39.063320300 -0600
@@ -14,7 +14,7 @@ libcrack_la_SOURCES = 	fascist.c \
 # For next ABI changing release, use 3:0:0
 # After that, follow the libtool recommended incrementing procedure
 #
-libcrack_la_LDFLAGS = -version-info 10:1:8
+libcrack_la_LDFLAGS = -version-info 10:1:8 -no-undefined
 
 # Link in NLS libs. Needed by FreeBSD build
 libcrack_la_LIBADD = $(LTLIBINTL)
--- origsrc/cracklib-2.8.15/python/Makefile.am	2009-11-18 17:58:20.000000000 -0600
+++ src/cracklib-2.8.15/python/Makefile.am	2010-02-16 14:26:39.289765000 -0600
@@ -2,7 +2,8 @@ if BUILD_PYTHON
 python_PYTHON = cracklib.py
 pyexec_LTLIBRARIES = _cracklibmodule.la
 AM_CFLAGS = -I$(top_srcdir)/lib
-_cracklibmodule_la_LDFLAGS = -module -avoid-version $(top_builddir)/lib/libcrack.la
+_cracklibmodule_la_LDFLAGS = -module -avoid-version -no-undefined
+_cracklibmodule_la_LIBADD = $(top_builddir)/lib/libcrack.la -lpython@PYTHON_VERSION@
 DEFS += '-DDEFAULT_CRACKLIB_DICT="$(DEFAULT_CRACKLIB_DICT)"'
 DEFS += '-DPYTHON_H="python@PYTHON_VERSION@/Python.h"'
 endif
