--- origsrc/cracklib-2.8.13/Makefile.am	2008-01-26 18:52:24.000000000 -0600
+++ src/cracklib-2.8.13/Makefile.am	2008-09-24 23:21:20.809880400 -0500
@@ -31,14 +31,14 @@
 
 dict: all
 	if test "x$(CROSS_COMPILING)" = "xno" ; then \
-		export PATH="$(top_builddir)/util:$$PATH" ; \
+		export PATH="$(top_srcdir)/util:$(top_builddir)/util:$(top_builddir)/lib/.libs:$$PATH" ; \
 		export LD_LIBRARY_PATH="$(top_builddir)/lib/.libs:$$LD_LIBRARY_PATH" ; \
 	fi ; \
 	create-cracklib-dict "$(srcdir)"/dicts/*
 
 dict-local: all
 	if test "x$(CROSS_COMPILING)" = "xno" ; then \
-		export PATH="$(top_builddir)/util:$$PATH" ; \
+		export PATH="$(top_srcdir)/util:$(top_builddir)/util:$(top_builddir)/lib/.libs:$$PATH" ; \
 		export LD_LIBRARY_PATH="$(top_builddir)/lib/.libs:$$LD_LIBRARY_PATH" ; \
 	fi ; \
 	cracklib-format "$(srcdir)"/dicts/* | cracklib-packer $(DESTDIR)/$(DEFAULT_CRACKLIB_DICT)
--- origsrc/cracklib-2.8.13/configure.in	2008-09-23 06:53:24.000000000 -0500
+++ src/cracklib-2.8.13/configure.in	2008-09-24 23:21:20.809880400 -0500
@@ -48,7 +48,7 @@
 AC_CHECK_FUNCS(getpwuid_r)
 
 dnl internationalization macros
-AM_GNU_GETTEXT_VERSION
+AM_GNU_GETTEXT_VERSION(0.17)
 AM_GNU_GETTEXT([external])
 
 dnl Control default dictname
