--- origsrc/cracklib-2.8.12/Makefile.am	2007-10-01 20:30:51.000000000 -0500
+++ src/cracklib-2.8.12/Makefile.am	2007-10-02 21:52:49.921875000 -0500
@@ -31,14 +31,14 @@
 
 dict: all
 	if test "x$(CROSS_COMPILING)" = "xno" ; then \
-		export PATH="$(top_builddir)/util:$$PATH" ; \
+		export PATH="$(top_srcdir)/util:$(top_builddir)/util:$(top_builddir)/lib/.libs:$$PATH" ; \
 		export LD_LIBRARY_PATH="$(top_builddir)/lib/.libs:$$LD_LIBRARY_PATH" ; \
 	fi ; \
 	create-cracklib-dict "$(srcdir)"/dicts/*
 
 dict-local: all
 	if test "x$(CROSS_COMPILING)" = "xno" ; then \
-		export PATH="$(top_builddir)/util:$$PATH" ; \
+		export PATH="$(top_srcdir)/util:$(top_builddir)/util:$(top_builddir)/lib/.libs:$$PATH" ; \
 		export LD_LIBRARY_PATH="$(top_builddir)/lib/.libs:$$LD_LIBRARY_PATH" ; \
 	fi ; \
 	cracklib-format "$(srcdir)"/dicts/* | cracklib-packer $(DESTDIR)/$(DEFAULT_CRACKLIB_DICT)
--- origsrc/cracklib-2.8.12/configure.in	2007-10-01 20:31:36.000000000 -0500
+++ src/cracklib-2.8.12/configure.in	2007-10-02 21:52:49.921875000 -0500
@@ -45,7 +45,7 @@
 AC_CHECK_FUNCS(getpwuid_r)
 
 dnl internationalization macros
-AM_GNU_GETTEXT_VERSION
+AM_GNU_GETTEXT_VERSION(0.14.5)
 AM_GNU_GETTEXT([external])
 
 dnl Control default dictname
--- origsrc/cracklib-2.8.12/lib/Makefile.am	2006-07-20 17:20:16.000000000 -0500
+++ src/cracklib-2.8.12/lib/Makefile.am	2007-10-02 21:52:49.921875000 -0500
@@ -14,7 +14,7 @@
 # For next ABI changing release, use 3:0:0
 # After that, follow the libtool recommended incrementing procedure
 #
-libcrack_la_LDFLAGS = -version-info 10:0:8
+libcrack_la_LDFLAGS = -version-info 10:0:8 -no-undefined
 
 # Link in NLS libs. Needed by FreeBSD build
 libcrack_la_LIBADD = $(LTLIBINTL)
--- origsrc/cracklib-2.8.12/python/Makefile.am	2006-02-28 20:13:52.000000000 -0600
+++ src/cracklib-2.8.12/python/Makefile.am	2007-10-02 21:52:49.953125000 -0500
@@ -1,6 +1,8 @@
 if BUILD_PYTHON
 pyexec_LTLIBRARIES = cracklibmodule.la
-cracklibmodule_la_LDFLAGS = -module -avoid-version -L$(top_builddir)/lib -lcrack
+cracklibmodule_la_LDFLAGS = -module -avoid-version -no-undefined
+cracklibmodule_la_LIBADD = $(top_builddir)/lib/libcrack.la \
+	-L/usr/lib/python$(PYTHON_VERSION)/config -lpython$(PYTHON_VERSION)
 DEFS += '-DDEFAULT_CRACKLIB_DICT="$(DEFAULT_CRACKLIB_DICT)"'
 DEFS += '-DPYTHON_H="python@PYTHON_VERSION@/Python.h"'
 endif
